org.eclipse.xtext.common.types.ui/src/org/eclipse/xtext/common/types/ui/notification/JavaBuilderState.java

```diff
@@ -223,7 +229,7 @@ private String getSimplePrimaryTypeName(ICompilationUnit compilationUnit) {
 		return elementName.substring(0, elementName.lastIndexOf("."));
 	}
 
-	private SimpleLookupTable getReferences() {
+	private Object getReferences() {
```

Comment:
I like that. Initially I though we'd have to dup the field, but having the two instanceof checks looks pretty clean - given the circumstances.